security.adoc

```diff
@@ -50,9 +50,15 @@ The community will fix security bugs for the latest major.minor version publishe
 |Latest 3.x
 |✅
 
+|3.15 LTS
+|✅
+
 |3.8 LTS
 |✅
 
+|3.2 LTS
```

Comment:
no, 3.2 LTS is not updated for security and bugs so should not be added.

Reply:
We need to remove that from https://github.com/quarkusio/quarkus/blob/main/SECURITY.md

Reply:
> no, 3.2 LTS is not updated for security and bugs so should not be added.

Removed 3.2 LTS :)